qiskit/pulse/__init__.py

```diff
@@ -74,6 +74,14 @@
    LoConfig
    LoRange
 
+Pulse Library
+=============
+
+.. autosummary::
+   :toctree: ../stubs/
+
+   qiskit.pulse.pulse_lib.discrete
```

Comment:
tilde works here too!
```suggestion
   ~qiskit.pulse.pulse_lib.discrete
```